docs/go/index.md

```diff
@@ -68,4 +68,4 @@ port := "8080"
  })
 ```
 
-Now wrap the routers, https clients and external dependencies like DBs. See the list of [supported frameworks](http://localhost:3000/docs/go/supported-frameworks).
+Now wrap the routers, https clients and external dependencies like DBs. See the list of [supported frameworks](/docs/go/supported-frameworks.md).
```

Comment:
.md in the end, is still causing a problem. 

Have you tried running/building the changes locally? 

Even the start-up logs are showing a warning. 
<img width="1174" alt="Screenshot 2022-09-20 at 10 56 42 AM" src="https://user-images.githubusercontent.com/15074229/191174555-3b3116ac-f228-45db-892d-566ac522c51d.png">

Reply:
I'll look into this and revert back.

Reply:
I tried running it locally, getting the same error but when I got rid of `.md` and made it a url link instead of a file one, the Deprecation Warning still exists. Tried many ways to resolve the same but no change. Although when the documentation website run on port 3000, the link is redirecting to supported frameworks but to a new tab.

Reply:
Yes, even if we convert the link to `docs.keplo.io/docs/go/supported-frameworks` it's opening in a new tab. And as it's a readme file, we can't open a link in the same tab.

A solution of this could be we can just put the supported framework topic next to the current topic if that doesn't make the flow improper